### PR TITLE
Dutch stuff

### PR DESCRIPTION
**Pull request by koeky**
Not butt stuff, as previously prophesized

### CODE REVIEW

**Comment by gdude2002** [comment]
@Marfjeh - Ding!

File: src/main/resources/translations/Messages_nl.properties [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+task_already_running = &cEr is al een taak bezig. Wacht alstublieft tot deze taak klaar is.
+unable_to_update_block_owner = Het is niet mogelijk om het eigenaar van de blokken te veranderen op %s(%s, %s, %s): %s
+unknown_player = &cOnbekende speler: &b%s
+unknown_world = &cOnbekende wereld: &b%s
+
+### COMMAND ALIASES ###
+# Command aliases may be split with a semicolon ";"
+# You can leave any of these commented out if you don't want to use them
+# Translators should provide localized command names as aliases
+
```
Comment (gdude2002) [inline src/main/resources/translations/Messages_nl.properties]: No command aliases

File: src/main/resources/translations/Messages_nl.properties [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+# &6> <player> &3| &b<player> &3| &b<player>
+# &6> <player> &3| &b<player> &3| &b<player>
+#
+# For the most parts, translators won't need to touch this, aside from translating "Friends"
+
+friends_command_list_header = == &d[&6Vrienden&d]&f ==
+friends_command_list_row = &b%s
+friends_command_list_separator = &3, &b
+
+friends_command_no_friends = &cGeen vrienden gevonden:(
```
Comment (gdude2002) [inline src/main/resources/translations/Messages_nl.properties]: Missing a space before the `:`

File: src/main/resources/translations/Messages_nl.properties [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+watchblock_import_fetched_uuid = UUID ophalen voor speler: %s
+watchblock_import_found_worlds = Gevonden werelden (%s): %s
+watchblock_import_friendships_converted = Vriendschappen overgezet.
+watchblock_import_importing_chunks = Overzetten segmenenten %s. Dit kan een tijdje duren.
+watchblock_import_loading_blocks_for_world = Blokken laden voor wereld: %s
+watchblock_import_looking_for_worlds = Zoeken voor werelden om te importeren…
+watchblock_import_not_all_worlds_converted = Niet alle werelden zijn overgezet. Gelieve te controleren op fouten!
+watchblock_import_setting_up_thread = Opstarten thread: %s
+watchblock_import_starting = WatchBlock importeren begint.
+watchblock_import_thread_completed = Thread voltooid: %s
```
Comment (gdude2002) [inline src/main/resources/translations/Messages_nl.properties]: Thread

File: src/main/resources/translations/Messages_nl.properties [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+watchblock_import_unable_to_fetch_uuid = UUID voor speler %s niet gevonden.
+watchblock_import_unable_to_find_data_files = Data bestanden niet gevonden!
+watchblock_import_unable_to_get_chunk_for_file = Segment voor bestand niet gevonden: %s
+watchblock_import_world_imported = Wereld geimporteerd :%s
```
Comment (gdude2002) [inline src/main/resources/translations/Messages_nl.properties]: `:` seems to be the wrong side of that space

File: src/main/resources/translations/Messages_nl.properties [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+watchblock_import_converting_friendships = Vriendschappen overzetten…
+watchblock_import_complete = Importeren is compleet!
+watchblock_import_created_friendship = Vrienschap gemaakt: %s -> %s
+watchblock_import_disabling_watchblock = WatchBlock wordt nu uitgeschakeld. Vergeet het niet om te verwijderen anders kunnen er volgende keer problemen voorkomen.
+watchblock_import_fetched_uuid = UUID ophalen voor speler: %s
+watchblock_import_found_worlds = Gevonden werelden (%s): %s
+watchblock_import_friendships_converted = Vriendschappen overgezet.
+watchblock_import_importing_chunks = Overzetten segmenenten %s. Dit kan een tijdje duren.
+watchblock_import_loading_blocks_for_world = Blokken laden voor wereld: %s
+watchblock_import_looking_for_worlds = Zoeken voor werelden om te importeren…
```
Comment (gdude2002) [inline src/main/resources/translations/Messages_nl.properties]: That's an ellipsis (`…`), not actually three dots. Make it three dots (`...`), please.

File: src/main/resources/translations/Messages_nl.properties [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+# Setowner command
+
+setowner_command_changing_ownership = &9Bezig met het veranderen van de eigenaar van de blokken &c%s&9. Dit kan een tijdje duren.
+setowner_command_make_worldedit_selection = &cMaak een geldige cuboid selectie alstublieft. 
+setowner_command_usage = &9Gebruik: &b/&3%s &2<speler>
+setowner_command_usage_exclamation_point = &9Plaats een uitroepteken &b"!"&9 inplaats van een gebruikersnaam / geselecteerde blokken om het te verstoten.
+
+# Transferblocks command
+
+transferblocks_command_transferring_blocks = &9Overzetten van Uw blokken naar &c%s&9. Dit kan een tijdje duren.
```
Comment (gdude2002) [inline src/main/resources/translations/Messages_nl.properties]: Should `Uw` be capitalised?

Comment (gdude2002) [inline src/main/resources/translations/Messages_nl.properties]: You didn't address this in your latest commit. Should it?

File: src/main/resources/translations/Messages_nl.properties [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+watchblock_import_fetched_uuid = UUID ophalen voor speler: %s
+watchblock_import_found_worlds = Gevonden werelden (%s): %s
+watchblock_import_friendships_converted = Vriendschappen overgezet.
+watchblock_import_importing_chunks = Overzetten segmenenten %s. Dit kan een tijdje duren.
+watchblock_import_loading_blocks_for_world = Blokken laden voor wereld: %s
+watchblock_import_looking_for_worlds = Zoeken voor werelden om te importeren...
+watchblock_import_not_all_worlds_converted = Niet alle werelden zijn overgezet. Gelieve te controleren op fouten!
+watchblock_import_setting_up_thread = Opstarten werk: %s
+watchblock_import_starting = WatchBlock importeren begint.
+watchblock_import_thread_completed = werk voltooid: %s
```
Comment (gdude2002) [inline src/main/resources/translations/Messages_nl.properties]: Capitalisation. `werk` -> `Werk`

**Comment by gdude2002** [comment]
This looks good to me, maybe @Marfjeh would like to have a look over it?

**Comment by Marfjeh** [comment]
Looks good! Nice job! :+1:

**Comment by gdude2002** [comment]
Are you happy with this, @Marfjeh, or do you want to merge some of your edits into this one?

**Comment by gdude2002** [comment]
\o/, thanks guys

**Pull request merged**